cob_hardware_config/cob3-2/config/base_controller.yaml

```diff
@@ -37,6 +37,10 @@ twist_controller:
 odometry_controller:
   type: cob_omni_drive_controller/OdometryController
   publish_rate: 50
+  frame_id: "/odom_combined"
```

Comment:
I guess we can get rid of the leading slashes here, as all calls to tf have the slash stripped under the hood, right?
Problems will probably only occur once we manipulate the respective ids manually.

Or not?